src/Controller/HearingBriefingController.php

```diff
@@ -211,6 +201,15 @@ private function createBriefingRecipients(array $parties, HearingBriefing $brief
             $briefingRecipient->setRecipient($recipient);
             $briefingRecipient->setHearingBriefing($briefing);
 
+            foreach ($briefing->getHearingPostRequest()->getHearingRecipients() as $hearingRecipient) {
+                // Do not add the document if current recipient is equal current hearing recipient.
```

Comment:
There's a typo here. The comment should read

```php
// Do not add the document if current recipient is equal to current hearing recipient.
```

or

```php
// Do not add the document if current recipient equals current hearing recipient.
```

Reply:
Fixed!